bin/llvm-krun

```diff
@@ -454,6 +466,10 @@ if $dryRun; then
   exit 0
 fi
 
+if [[ "$OSTYPE" == "darwin"* ]]; then
+  replace_gdb_by_lldb
```

Comment:
I would prefer to have the command chosen up front, rather than performing string replacements after the fact, I think